.github/workflows/cmake-ctest.yml

```diff
@@ -157,7 +157,7 @@ jobs:
           endpoint: ${{ secrets.AZURE_ENDPOINT }}
           trusted-signing-account-name: ${{ secrets.AZURE_CODE_SIGNING_NAME }}
           certificate-profile-name: ${{ secrets.AZURE_CERT_PROFILE_NAME }}
-          files-folder: ${{ runner.workspace }}/hdf5116/build/${{ inputs.preset_name }}-MSVC
+          files-folder: ${{ runner.workspace }}/hdf5/build116/${{ inputs.preset_name }}-MSVC
```

Comment:
What was the motivation for changing the build folder names to `build116`? Seems like another maintenance headache and since these build shouldn't ever interact with develop ones it seems like we could leave them as `build` to not have to change that.